website/content/docs/connect/config-entries/proxy-defaults.mdx

```diff
@@ -275,8 +275,7 @@ spec:
         Envoy will expose listeners for these checks and will only accept connections originating from localhost or Consul's
         [advertise address](/docs/agent/options#advertise). The port for these listeners are dynamically allocated from
```

Comment:
I removed the kube part of this since we handle that automatically for kube users now.